makeWorkspace/make_ws.py

```diff
@@ -31,6 +31,85 @@ def cli_args():
 
     return args
 
+def create_workspace(fin, f_jes, fout, category):
```

Comment:
I have not reviewed this line by line, but it's just a straightforward refacotring of the code that was previously in the main function?

Reply:
Yes, it's essentially a copy-paste of what was inside the main function earlier. I just thought this way the code is a bit easier to follow